app/views/projects/samples/_table.html.erb

```diff
@@ -2,6 +2,7 @@
   id="samples-table"
   class="min-w-full table-fixed dark:divide-slate-600"
   data-controller="selection"
+  data-selection-action-link-outlet=".link"
```

Comment:
Can we update this to `.action-link` so it's more specific to what its being used for (example here..action)

Reply:
Updated in 6777432a27326c3d09e2af26cee263544ca2bc18